src/components/core/Text/Text.tsx

```diff
@@ -1,8 +1,8 @@
 import styled from 'styled-components';
 import colors from '../colors';
 
-const TITLE_FONT_FAMILY = 'GT Alpina';
-export const BODY_FONT_FAMILY = 'ABC Diatype';
+const TITLE_FONT_FAMILY = "'GT Alpina', serif";
+export const BODY_FONT_FAMILY = "'ABC Diatype', Helvetica, Arial, sans-serif";
```

Comment:
In addition to adding fallbacks, we also wrap the primary font in quotation marks. Although functionally this shouldn't make a difference, the [spec])https://www.w3.org/TR/CSS2/fonts.html#font-family-prop) does  recommend wrapping fonts with spaces in quotation marks:

> To avoid mistakes in escaping, it is recommended to quote font family names that contain white space, digits, or punctuation characters other than hyphens:
`body { font-family: "New Century Schoolbook", serif }`

[more fun reading](https://mathiasbynens.be/notes/unquoted-font-family)

Reply:
sick! great research